src/app/accounts/active-accounts.ts

```diff
@@ -16,7 +16,7 @@ export const getRecentlyActiveAccounts = async function* ():
   const unlockedAccounts = AccountsRepository().listUnlockedAccounts()
   if (unlockedAccounts instanceof Error) return unlockedAccounts
 
-  const displayPriceRatio = await getCurrentPriceAsPriceRatio({
+  const displayPriceRatio = await getCurrentPriceAsWalletPriceRatio({
```

Comment:
if this is not using `getCurrentPriceAsDisplayPriceRatio` then this should be renamed

Reply:
https://github.com/GaloyMoney/galoy/pull/2334#discussion_r1108847733